templates/accounts.go

```diff
@@ -26,58 +26,126 @@ import (
 )
 
 const createAccountTemplate = `
-transaction(publicKeys: [[UInt8]], code: [UInt8]) {
+transaction(publicKeys: [[UInt8]]) {
   prepare(signer: AuthAccount) {
 	let acct = AuthAccount(payer: signer)
 
 	for key in publicKeys {
 	  acct.addPublicKey(key)
 	}
-	
-	if code.length > 0 {
-	  acct.setCode(code)
+  }
+}
+`
+
+const createAccountWitContractsTemplate = `
+transaction(publicKeys: [[UInt8]], contracts: {String: [UInt8]}) {
+  prepare(signer: AuthAccount) {
+	let acct = AuthAccount(payer: signer)
+
+	for key in publicKeys {
+	  acct.addPublicKey(key)
+	}
+
+    for contract in contracts.keys {
+	  acct.contracts.add(name: contract, code: contracts[contract]!)
```

Comment:
Would be nice to add key-value iteration over dictionaries to make this more ergonomic

Reply:
There should also be an example of iterating over a dictionary in the docs. I can add that. :)

Reply:
Oh, this was more like an observation/note for myself! Cadence doesn't support that yet, it would be nice to add it there 😅

Reply:
What I meant is that it took me a bit to figure this out 😄 :
```
    for contract in contracts.keys {
	  acct.contracts.add(name: contract, code: contracts[contract]!)
    }
```

As it is not mentioned in the docs right now.

Reply:
Oh, I misunderstood! Good point, I added an example to the documentation here: https://github.com/onflow/cadence/pull/399